samples/ants/2.1.0rc1/Dockerfile.centos5

```diff
@@ -0,0 +1,44 @@
+# Build ANTs 2.1.0rc1
```

Comment:
I agree. But I will leave the RC hashes in the ANTs class so RCs can be built from source.